### PR TITLE
Describe ArduSub Automatic motor direction detection feature

### PR DESCRIPTION
**Pull request by Williangalvani**
docs for https://github.com/mavlink/qgroundcontrol/pull/8236

Not sure if if should put this too under the daily releases new features

### CODE REVIEW

File: en/SetupView/Motors_ardusub.md [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 
 ![Ardusub Motors Test](../../assets/setup/motors-sub.jpg)
+
+## Automatic Direction Detection
+
+Ardusub 4.0 and newer support automatic detection of the motor directions.
+This works by applying pulses to each motor, checking if the frame reacts as expected, and reversing the motor if necessary.
+The process takes around one minute.
+
+To perform the automatic motor direction detection, click the "Auto-Detect Directions" button and wait.
```
Comment (jaxxzer) [inline en/SetupView/Motors_ardusub.md]:
```suggestion
To perform the automatic motor direction detection, navigate to 
_Vehicle Setup_->_Motors_ tab, and click the "Auto-Detect Directions" button and wait.
```

File: en/SetupView/Motors_ardusub.md [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
 
 ![Ardusub Motors Test](../../assets/setup/motors-sub.jpg)
+
+## Automatic Direction Detection
+
+Ardusub 4.0 and newer support automatic detection of the motor directions.
+This works by applying pulses to each motor, checking if the frame reacts as expected, and reversing the motor if necessary.
+The process takes around one minute.
+
+To perform the automatic motor direction detection, navigate to _Vehicle Setup_->_Motors_ tab, click the "Auto-Detect Directions" button and wait.
```
Comment (hamishwillee) [inline en/SetupView/Motors_ardusub.md]: I tend to do button clicks and navigation as bold like this:

> To perform the automatic motor direction detection, navigate to **Vehicle Setup > Motors** tab, click the **Auto-Detect Directions** button and wait.

```
To perform the automatic motor direction detection, navigate to **Vehicle Setup > Motors** tab, click the **Auto-Detect Directions** button and wait.
```

File: en/SetupView/Motors_ardusub.md [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
 If the frame selection does not match your vehicle, first select the correct frame in the [Frame](../SetupView/airframe_ardupilot.md#ardusub) tab.
 
-To set up and test the motors, read and follow the instructions on the page.
+To manually set up and test the motors, read and follow the instructions on the page.
 
 > **Warning** Make sure the motors and propellers are clear from obstructions before sliding the switch to arm the vehicle and enable the test!
 
 ![Ardusub Motors Test](../../assets/setup/motors-sub.jpg)
+
+## Automatic Direction Detection
```
Comment (hamishwillee) [inline en/SetupView/Motors_ardusub.md]: What do you want people to do? If the automatic testing is the recommended and easiest approach then suggest you put docs for it first.

I also quite like direction at the top - e.g below main heading tell people their options and why each might be selected.

Comment (Williangalvani) [reply en/SetupView/Motors_ardusub.md]: Thank you for the tips.
I added a section below the main heading explaining how it works.

**Comment by hamishwillee** [comment]
THanks for fixes. I'll review again when this comes out of draft.

**Pull request merged**